scripts/eslint-pre-commit

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
-
-git stash -q --keep-index
-
-# Test prospective commit
-git diff-index --cached HEAD --name-only --diff-filter ACMR | egrep '.js$' | xargs -I $(npm bin)/eslint {}
```

Comment:
Don't know how to setup up an organization account on Twitter, but stuff like this could be published there.